<!-- 
This launch file launches the required ros nodes to get the speech recognition and
behavior execution working.
-->

<launch>
	<arg name="nao_ip" default="$(optenv NAO_IP 127.0.0.1)" />
  	<arg name="nao_port" default="$(optenv NAO_PORT 9559)" />

	<node name="nao_behaviors" pkg="nao_driver" type="nao_behaviors.py" required="true" args="--pip=$(arg nao_ip) --pport=$(arg nao_port)"/>
	<node name="nao_speech" pkg="nao_driver" type="nao_speech.py" required="true" args="--pip=$(arg nao_ip) --pport=$(arg nao_port)"/>
</launch>
